src/Compilers/VisualBasic/Test/Semantic/FlowAnalysis/ImplicitVariableTests.vb

```diff
@@ -46,6 +46,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.FlowAnalysis
             ]]>,
             alwaysAssigned:={"f", "x"},
             captured:={"x"},
+            capturedInside:={"x"},
```

Comment:
Just to confirm: this test would fail without the added line, correct?
Could you add a test to exercise `capturedOutside` too?

Reply:
Yes, absent of the argument asserts for an empty set.

Test added.